src/components/Inputs/Inputs.tsx

```diff
@@ -0,0 +1,103 @@
+import {useNavigation} from '@react-navigation/native';
+import React, {useRef, useState} from 'react';
+import {Image, Pressable, Text, TextInput, View} from 'react-native';
+import {Images} from '../../assets/images';
+import {colors} from '../../constants/colors';
+import {styles} from './Inputs-styles';
+
+export function RCalTextInput({
+  label,
+  inputValue,
+  placeholder,
+  onChangeHandler,
+  errMessage,
+}: {
+  label: string;
```

Comment:
I am guessing we are combining the list of all inputs and buttons in one file structure, is this common practice?

Reply:
I am not entirely sure if this is a common practice. But I have seen it in some repos. If you think it prudent, I can create separate files for each input and each button.